app/assets/sass/tai05102018.scss

```diff
@@ -70,3 +70,7 @@ hr {
     word-break: break-word;
   }
 }
+
+.validation-error__inline-error {
```

Comment:
This needs media query
font-size: 16px;
@media (min-width: 641px) {
    font-size: 19px;
}